README.md

```diff
@@ -184,6 +177,27 @@ badgeView.colorScheme = .static(.blackOnMint)
 badgeView.colorScheme = .dynamic(lightPalette: .whiteOnBlack, darkPalette: .blackOnWhite)
 ```
 
+### Badge
+
+The Afterpay badge is a simple `UIView` that can be scaled to suit the needs of your app. As per branding guidelines it has a minimum width constraint of 64 points.
+
+```swift
+let badgeView = BadgeView()
+```
+
+Below are examples of the badge in each of it's color schemes:  
```

Comment:
It might be nice to align the wording of this with the payment button below.

Should it be "it's colour schemes" or "the color schemes"?